conjure-undertow-processor/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'com.palantir.external-publish-jar'
+apply plugin: 'org.revapi.revapi-gradle-plugin'
```

Comment:
This is an annotation processor, the annotations themselves must be compatible, but not the processor implementatiob